### PR TITLE
Use BlobBuilder instead of Blob constructor in older Browsers (fixes #843)

### LINKED ISSUE #843: "Illegal constructor" in older Android Browsers
Seeing this error when trying to create an instance of `Blob` with `new Blob()`.

### PR DESCRIPTION
**Pull request by galfert**
This fix prevents the `Illegal constructor` error for Browsers that don't support the Blob constructor.

### CODE REVIEW

**Comment by skddc** [comment]
Wow, that hound can be annoying. Looks like there's exactly one line with double quotes at the beginning of the file. ;)

**Comment by galfert** [comment]
Ok, I hope I made dem hounds happy now :)

**Comment by skddc** [comment]
+1

File: src/wireclient.js [hunk 1]
```diff
@@ -93,10 +93,19 @@
   }
 
   function readBinaryData(content, mimeType, callback) {
-    var blob = new Blob([content], { type: mimeType });
+    var blob;
+    global.BlobBuilder = global.BlobBuilder || global.WebKitBlobBuilder;
```
Comment (silverbucket) [inline src/wireclient.js]: Does the resulting var need to be global? This *might* cause issues with other libs, no?

Comment (skddc) [inline src/wireclient.js]: Good point. Albeit if it re-assigns it to the prefixed one, there's no harm really, because it's the same thing.

**Comment by skddc** [comment]
@silverbucket Is this +1 otherwise?

**Comment by silverbucket** [comment]
Yup, +1

**Comment by skddc** [comment]
Cool, thanks!

**Pull request merged**